datasets/coco.py

```diff
@@ -161,9 +167,12 @@ def build(image_set, args):
     PATHS = {
         "train": (root / "train2017", root / "annotations" / f'{mode}_train2017.json'),
         "val": (root / "val2017", root / "annotations" / f'{mode}_val2017.json'),
+        "train_combined": (root / "train2017", root / "annotations" / f'{mode}_train_combined.json'),
+        "train_combined_nms": (root / "train2017", root / "annotations" / f'{mode}_train_combined.json')
```

Comment:
Let's have different `.json` file names for these two settings. Maybe `*_train_combined.json` and `*_train_combined_nms.json`